spec/rubocop/cop/rails/pick_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::Pick, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'when using Rails 6.0 or newer', :rails60 do
+    context 'with one argument' do
+      it 'registers an offense for `pluck(...).first' do
+        expect_offense(<<~RUBY)
+          x.pluck(:a).first
+          ^^^^^^^^^^^^^^^^^ Prefer `pick` over `pluck(...).first`.
```

Comment:
I think the warning can show specific values such as:

```suggestion
          ^^^^^^^^^^^^^^^^^ Prefer `pick(:a)` over `pluck(:a).first`.
```

Reply:
It would be better if the warning range could be specified.

```diff
-          x.pluck(:a).first
-          ^^^^^^^^^^^^^^^^^ Prefer `pick` over `pluck(...).first`.
+          x.pluck(:a).first
+            ^^^^^^^^^^^^^^^ Prefer `pick` over `pluck(...).first`.
```

Reply:
Both great suggestions, thanks! I pushed an update.